x-pack/plugins/security_solution/server/endpoint/routes/agent/agent_status_handler.test.ts

```diff
@@ -43,60 +64,55 @@ describe('Agent Status API route handler', () => {
     apiTestSetup.endpointAppContextMock.experimentalFeatures = {
       ...apiTestSetup.endpointAppContextMock.experimentalFeatures,
       responseActionsSentinelOneV1Enabled: true,
-      responseActionsCrowdstrikeManualHostIsolationEnabled: false,
-      agentStatusClientEnabled: false,
+      responseActionsCrowdstrikeManualHostIsolationEnabled: true,
     };
 
     registerAgentStatusRoute(apiTestSetup.routerMock, apiTestSetup.endpointAppContextMock);
   });
 
-  it('should error if the sentinel_one feature flag is turned off', async () => {
-    apiTestSetup.endpointAppContextMock.experimentalFeatures = {
-      ...apiTestSetup.endpointAppContextMock.experimentalFeatures,
-      responseActionsSentinelOneV1Enabled: false,
-      responseActionsCrowdstrikeManualHostIsolationEnabled: false,
-    };
+  it.each`
```

Comment:
Oh nice I didn't know you can do this for `each` instead of passing in an array!! This is so much nicer to read and maintain. 🚀

Reply:
yeah... it works well when the values are not too long